### ISSUE TITLE
PlayerDeathEvent not working properly

### ISSUE THREAD

**Issue by TeamRaiderz** [teozfrank/DuelMe#76]
Hello! 
We found a bug in your DuelMe plugin. The error is that when a player is killed in a duel, the duel will not remove the players from the duel and the duel's timer will go on.  I tried to fix it myself with my plugin just using your plugin's DuelManager class, but with no success (I just checked in my DeathEvent if the players were in the duel, and then remove them from the duel). I think the error is at PlayerDeath.java class at the line 78 (dm.endDuel(loser);). I don't know if this has occured for others or I don't know if my PvPMoney plugin or my KitPvP plugin for the server is not working well with your plugin, but it would be nice if you could try to look at the error.
Our server has DuelMe plugin with version 0.17.2
Thanks!
 ~ TeamRaiderz

**Comment by teozfrank** [comment]
Do you have any combat log plugins? they can cause a conflict causing players to not be teleported from an arena. Could you show me this bug on your server?

**Comment by TeamRaiderz** [comment]
Here is a video what shows the bug: https://www.youtube.com/watch?v=j4S8ktSHSPI&feature=youtu.be

We do not have any combat log plugins.. We just have a "PvPMoney" plugin what gives the player money after the player kills another player. And I've also made a KitPvP plugin as you probably saw in the video. They both contain a PlayerDeathEvent, so I checked in both plugins if the player is in a duel if they were, the plugin would end the duel.
